src/steps/tiltakssteg/Tiltakssteg.tsx

```diff
@@ -46,30 +40,29 @@ export default function Tiltakssteg({ title, stepNumber, onCompleted, onGoToPrev
             onGoToPreviousStep={onGoToPreviousStep}
             guide={<Veiledningstekst brukerHarRegistrerteTiltak={brukerHarRegistrerteTiltak} />}
             submitSectionRenderer={submitSectionRenderer}
-            hideStepIndicator={!brukerHarRegistrerteTiltak}
+            hideStepIndicator={!brukerHarRegistrerteTiltakMedGyldigPeriode}
             hideTitle={!brukerHarRegistrerteTiltak}
         >
-            {brukerHarRegistrerteTiltak && (
+            {brukerHarRegistrerteTiltakMedUgyldigPeriode && (
+                <Alert variant="warning" style={{ marginTop: '1rem', marginBottom: '2rem' }}>
+                    <List as="ul" title="Vi har funnet tiltak som mangler start- eller sluttdato">
+                        {tiltakMedUgyldigPeriode.map((t) => (
+                            <List.Item key={t.aktivitetId}>{lagTiltaksalternativTekst(t)}</List.Item>
+                        ))}
+                    </List>
+                    Dersom du ønsker å søke tiltakspenger for et av disse tiltakene, må du sende søknad på
+                    papir eller kontakte oss slik at vi kan registrere fullstendig periode på tiltaket.
+                </Alert>
+            )}
+            {brukerHarRegistrerteTiltakMedGyldigPeriode && (
                 <Flervalgsspørsmål
-                    alternativer={tiltak.map(lagSvaralternativForTiltak)}
+                    alternativer={tiltakMedGyldigPeriode.map(lagSvaralternativForTiltak)}
                     name="svar.tiltak.aktivitetId"
                     validate={valgtTiltakValidator}
                 >
                     Hvilket tiltak ønsker du å søke tiltakspenger for?
                 </Flervalgsspørsmål>
             )}
-            {(brukerHarValgtEtTiltak && valgtTiltakManglerHelePerioden ) && (
-                <TiltakMedUfullstendigPeriodeUtfylling
```

Comment:
Brukes ikke disse komponentene lengre (`TiltakMedUfullstendigPeriodeUtfylling` og `TiltakMedFullstendigperiodeUtfylling`)? Isåfall kan vel de fjernes

Reply:
Tenker vi kan beholde dem, i fall vi trenger denne funksjonaliteten.